superset-frontend/src/components/Badge/index.tsx

```diff
@@ -22,7 +22,7 @@ import { styled } from '@superset-ui/core';
 import { Badge as AntdBadge } from 'antd';
```

Comment:
You can remove this comment about `no-restricted-imports` now because we have a `.eslintrc` in the components folder allowing it.